### ISSUE TITLE
view inside popup doesn't get initialized

### ISSUE THREAD

**Issue by ymilhahn** [framework7io/Framework7-Vue#243]
Hey everyone!

Since the update to 0.9.3 the following setup is not working anymore.

## MARKUP

```
<div id="app">

	<f7-views>
		<f7-view id="main-view" main>
			<f7-pages>
			</f7-pages>
		</f7-view>
	</f7-views>

	<f7-popup id="my-popup">
		<f7-view id="my-view">
		</f7-view>
	</f7-popup>

</div>
```
## EXPECTED RESULT

Before 0.9.3 the view (#my-view) was initialized properly and became part of the DOM:

```
<div class="popup modal-out" id="my-popup">
	<div class="view navbar-fixed navbar-through" id="my-view">
		<div class="pages">
		</div>
	</div>
</div>
```

The popup and view could be used via:
```
<f7-link open-popup="#my-popup" view="#my-view" href="/mypage">Open Page</f7-link>
```

## ACTUAL RESULT

Since 0.9.3 the view only gets rendered in the DOM when `:opened="true"`. Otherwise the resulting HTML is this:
```
<div class="popup modal-out" id="my-popup">
<!---->
</div>
```

Trying to open a page in the view (#my-view) then results in

```Uncaught TypeError: Cannot read property 'f7View' of undefined```

since the HTML-Element can not be found.

---

Do I have to change my markup or is something broken since the last version?
Thanks in advance!

**Comment by Robula** [comment]
I'm having a similar issue. 
### Markup
```
<div id="app">
	<f7-views>
		<f7-view id="main-view" main>
			<f7-pages>
                <f7-page>
                    <f7-link open-popup="#my-popup">Open Popup</f7-link>
                </f7-page>
			</f7-pages>
		</f7-view>
	</f7-views>
	<f7-popup id="my-popup">
            <h1>Content</h1>
	</f7-popup>
</div>
```
I can open the popup but the popup is completely empty. If I inspect the DOM I see this:
```
<div data-v-015e3f3c="" data-v-205fdaf2="" class="popup modal-in" id="my-popup" opened="true">
    <!---->
</div>
````

**Comment by Robula** [comment]
Is this something to do with 5115ed0c0191938b1290a57c06161c030250e393?

**Comment by levischuckeats** [comment]
I confirm that rolling back to `0.9.2` resolved this regression. 

This is a pretty significant feature to be broken.

**Comment by scriptPilot** [comment]
Same for me - Popup views are initialized if the popup is visible only.

Workaround:

```
<f7-popup style="left: 100%" opened>
  <f7-view url="..." />
</f7-popup>

...

mounted() {
  this.$f7.closeModal('.popup')
  this.$$('.popup').css('left', '0')
}
```

**Comment by meteozond** [comment]
Looks like something completely wrong with `f7-popup`.
`f7-navbar` inside `f7-popup` component covers the content, but it's ok with legacy `div` notation.
![](https://i.gyazo.com/b4d82d5754b139512347db14a27509d2.png)

**Comment by decates** [comment]
@Robula is right. The fix #232 has broken the popup. I've asked @nolimits4web if we can roll back that change for now, and look at fixing that issue a different way.

**Comment by JCKodel** [comment]
Any news on that? Popups are completely broken (even in 0.9.2). For me, 0.9.2 works fine, but only once. When I close and reopen it, something weird occurs:

This is the DOM when the popup is first opened (the yellow one is the popup, on its own view):

![image](https://user-images.githubusercontent.com/379339/33450372-b83ac57c-d5f2-11e7-976e-4c7d07db83a3.png)

The red marking shows the correct page (beneath the popup). Everything working just fine.

Now, I close the popup:

![image](https://user-images.githubusercontent.com/379339/33450420-e226e3d4-d5f2-11e7-8cd7-a21797a27fbd.png)

The popup no longer is in DOM, everything is fine.

Now, I try to navigate to the next page (this new page doesn't have popup):

![image](https://user-images.githubusercontent.com/379339/33450494-1bd51e70-d5f3-11e7-87e5-c7eda987df31.png)

The red one is the correct new page, but, look! My popup is inserted into the navigation >.< In fact, it is shown instead of correct page! The weird part is that this happens on the main view (not the popupview). =\

Here's the code to the page that contains the popup:
```
<template>
    <f7-page name="capillaryScheduleIndex" pull-to-refresh @ptr:refresh="loadData">
        <f7-navbar title="⏰ Cronograma Capilar" back-link="Voltar" sliding>
            <f7-nav-right>
                <f7-link open-panel="right"><f7-badge color="pink">💎 {{ $store.state.userInfo.diamonds }}</f7-badge></f7-link>
            </f7-nav-right>
        </f7-navbar>

        <f7-block-title v-if="schedules.length > 0">Seus cronogramas:</f7-block-title>

        <f7-list media-list v-if="schedules.length > 0">
            <f7-list-item swipeout :link="`/capillarySchedule/details/${item.id}`" v-for="item in schedules" :key="item.id" @swipeout:deleted="deleteItem(item)">
                <div slot="inner">
                    <div class="item-title-row">
                        <div :class="scheduleNameClass(item)">{{ scheduleName(item) }}</div>
                    </div>

                    <div class="item-subtitle" v-html="schedulePeriod(item)"></div>
                </div>

                <f7-swipeout-actions>
                    <f7-swipeout-button delete data-confirm="Tem certeza de que deseja excluir este cronograma?" data-confirm-title="Excluir cronograma" data-close-on-cancel="true">Excluir</f7-swipeout-button>
                </f7-swipeout-actions>
            </f7-list-item>

            <f7-list-label>Arraste o item para esquerda para excluir</f7-list-label>
        </f7-list>

        <f7-block-title v-if="schedules.length === 0">Você não tem cronogramas ainda</f7-block-title>
        <f7-block v-if="schedules.length === 0">Toque no botão <f7-icon material="add_circle" class="inner-icon" /> para responder ao quiz e criar um novo cronograma para teus cabelos.</f7-block>

        <f7-fab :class="schedules.length === 0 ? 'action-call' : ''" @click="createNewSchedulePopup">
            <f7-icon material="add" />
        </f7-fab>

        <f7-popup id="createNewSchedulePopup">
            <f7-view name="createNewSchedulePopup" navbar-through toolbar-through :dynamic-navbar="true" />
        </f7-popup>
    </f7-page>
</template>

<script src="./index.js" />
<style src="./index.scss" lang="scss" />
```
(last element of page).

The popup is displayed using this code:

```
createNewSchedulePopup()
{
	this.$store.dispatch("resetCapillaryScheduleSteps");
	this.$f7.views["createNewSchedulePopup"].router.load({ url: "/capillarySchedule/steps/index", animatePages: false });

	const popup = window.Dom7("#createNewSchedulePopup");

	window.f7.popup(popup);

	popup.once("popup:closed", () =>
	{
		this.loadData();
	});
}
```

And closed by VUEX (payload is "#createNewSchedulePopup"):
```
confirmPopupClose({ commit }, payload)
{
	window.f7.confirm(payload.text, payload.title, () =>
	{
		window.f7.closeModal(window.Dom7(payload.popup));
	});
},
```

The normal navigation is made through normal href links, nothing special. :(
